gui/src/main/java/com/devonfw/ide/gui/AppLauncher.java

```diff
@@ -0,0 +1,31 @@
+package com.devonfw.ide.gui;
+
+public class AppLauncher {
```

Comment:
What is the purpose of this class? IMHO you should just put a regular `main` method (without creating your own `Thread`) into `App` like suggested in every JavaFx tutorial.
Maybe the CobiGen example that Jan showed you was rather misleading than helpful...

Reply:
Running the main within App does not work for me, it throws an Error saying JavaFX runtime components are missing. Calling the App from the Launcher however works. 

Seems to be a common solution for the problem: https://stackoverflow.com/questions/56894627/how-to-fix-error-javafx-runtime-components-are-missing-and-are-required-to-ru

Reply:
I am totally puzzled that another `main` method delegating to the actual `App.main` method solves such error but I pulled your feature-branch and can confirm this strange effect.
Since I was using JPMS with my other project using JavaFx I never have seen this bug since the actual root of the bug is the java module system and using `add opens` directives the error can also be worked around.
Most probably with Eclipse and EclipseFx Plugin it would also work but since we are using IntelliJ let us keep this workaround unless we may find an even better solution.
Since you nicely reworked and simplified this the extra `AppLauncher` class does not hurt or seem like any maintenance overhead so thanks for your solution.
What you however should do is always adding JavaDoc and here from the JavaDoc you could link the stack-overflow discussion so also in the future when this PR is history and forgotten developers can still understand what this is all about.